File: Ruby/Transaction/Card Transaction.rb
require 'net/http'
require 'uri'

uri = URI.parse("https://api.juspay.in/txns")
request = Net::HTTP::Post.new(uri)
request.content_type = "application/x-www-form-urlencoded"
request["Accept"] = "*/*"
request["Host"] = "api.juspay.in"

request.set_form_data(
  "card_exp_month" => "07",
  "card_exp_year" => "22",
  "card_number" => "5419190202700020",
  "card_security_code" => "015",
  "format" => "json",
  "merchant_id" => "guest",
  "name_on_card" => "test",
  "order_id" => "ORD_1557929645",
  "payment_method" => "MASTERCARD",
  "payment_method_type" => "CARD",
  "redirect_after_payment" => "true",
  "save_to_locker" => "true",
)

req_options = {
  use_ssl: uri.scheme == "https",
}

response = Net::HTTP.start(uri.hostname, uri.port, req_options) do |http|
    response = http.request(request)
    puts response.read_body
end

# response.code
# response.body

# {"txn_uuid":"1jzverxzmnaovugq","txn_id":"guest-ORD_1557929645-7","status":"PENDING_VBV","payment":{"authentication":{"url":"https://api.juspay.in/pay/start/guest/1jzverxzmnaovugq","method":"GET"}},"order_id":"ORD_1557929645"}

File: PHP/Transactions/CardTransaction.php
<?php

$curl = curl_init();
$order_id="ORD_1557929645";
$merchant_id="aman_prod";

curl_setopt_array($curl, array(
  CURLOPT_URL => "https://api.juspay.in/txns",
  CURLOPT_RETURNTRANSFER => true,
  CURLOPT_ENCODING => "",
  CURLOPT_MAXREDIRS => 10,
  CURLOPT_TIMEOUT => 30,
  CURLOPT_HTTP_VERSION => CURL_HTTP_VERSION_1_1,
  CURLOPT_CUSTOMREQUEST => "POST",
  CURLOPT_POSTFIELDS => "------WebKitFormBoundary7MA4YWxkTrZu0gW\r\nContent-Disposition: form-data; name=\"order_id\"\r\n\r\n$order_id\r\n------WebKitFormBoundary7MA4YWxkTrZu0gW\r\nContent-Disposition: form-data; name=\"merchant_id\"\r\n\r\n$merchant_id\r\n------WebKitFormBoundary7MA4YWxkTrZu0gW\r\nContent-Disposition: form-data; name=\"payment_method_type\"\r\n\r\nCARD\r\n------WebKitFormBoundary7MA4YWxkTrZu0gW\r\nContent-Disposition: form-data; name=\"payment_method\"\r\n\r\nMASTERCARD\r\n------WebKitFormBoundary7MA4YWxkTrZu0gW\r\nContent-Disposition: form-data; name=\"card_number\"\r\n\r\n5419190202700020\r\n------WebKitFormBoundary7MA4YWxkTrZu0gW\r\nContent-Disposition: form-data; name=\"card_exp_year\"\r\n\r\n22\r\n------WebKitFormBoundary7MA4YWxkTrZu0gW\r\nContent-Disposition: form-data; name=\"card_exp_month\"\r\n\r\n07\r\n------WebKitFormBoundary7MA4YWxkTrZu0gW\r\nContent-Disposition: form-data; name=\"name_on_card\"\r\n\r\ntest\r\n------WebKitFormBoundary7MA4YWxkTrZu0gW\r\nContent-Disposition: form-data; name=\"card_security_code\"\r\n\r\n015\r\n------WebKitFormBoundary7MA4YWxkTrZu0gW\r\nContent-Disposition: form-data; name=\"save_to_locker\"\r\n\r\ntrue\r\n------WebKitFormBoundary7MA4YWxkTrZu0gW\r\nContent-Disposition: form-data; name=\"redirect_after_payment\"\r\n\r\ntrue\r\n------WebKitFormBoundary7MA4YWxkTrZu0gW\r\nContent-Disposition: form-data; name=\"format\"\r\n\r\njson\r\n------WebKitFormBoundary7MA4YWxkTrZu0gW--",
  CURLOPT_HTTPHEADER => array(
    "Accept: */*",
    "Authorization: Basic MTExMTExMTExMTExMTExMTExMTExMTExMTExOg==",
    "Cache-Control: no-cache",
    "Connection: keep-alive",
    "Host: api.juspay.in",
    ": 733cf7da-910f-4505-9ebb-412ca09508a5,4945d6cb-bfa6-41db-bd48-81bc861f9fc2",
    "User-Agent: PostmanRuntime/7.11.0",
    "accept-encoding: gzip, deflate",
    "cache-control: no-cache",
    "content-length: 1482",
    "content-type: multipart/form-data; boundary=----WebKitFormBoundary7MA4YWxkTrZu0gW"
  ),
));

$response = curl_exec($curl);
$err = curl_error($curl);

curl_close($curl);

if ($err) {
  echo "cURL Error #:" . $err;
} else {
  echo $response;
}

// Response 
// { "order_id": "ORD_1557929645",
//    "txn_id": "aman_prod-ORD_1557929645-2",
//    "txn_uuid": "bar4whobdudkiopg",
//    "status": "PENDING_VBV",
//    "payment": { "authentication": { "method": "GET",
//    "url": "https://api.juspay.in/pay/start/aman_prod/bar4whobdudkiopg" } } }